data/talkactions/scripts/reload.lua

```diff
@@ -8,9 +8,6 @@ local reloadTypes = {
 	["channel"] = { targetType = RELOAD_TYPE_CHAT, name = "chatchannels" },
 	["chatchannels"] = { targetType = RELOAD_TYPE_CHAT, name = "chatchannels" },
 
-	["command"] = { targetType = RELOAD_TYPE_COMMANDS, name = "commands" },
-	["commands"] = { targetType = RELOAD_TYPE_COMMANDS, name = "commands" },
-
```

Comment:
It's because he merged instead of rebasing. It won't count as changed lines if merged, but it's ugly anyway.